api-reference/50 Common/utils/viz/registerGradient(type_options).md

```diff
@@ -5,22 +5,185 @@ export: registerGradient
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Registers a new gradient.
 
 ##### return: String
-<!-- Description goes here -->
+The fill id.
 
 ##### param(type): String
-<!-- Description goes here -->
+The gradient type: 'linear' or 'radial'.
 
 ##### param(options): Object
-<!-- Description goes here -->
+The gradient's settings.
 
 ##### field(options.colors): Array<GradientColor>
-<!-- Description goes here -->
+The array of gradient colors.
 
 ##### field(options.rotationAngle): Number
-<!-- Description goes here -->
+The rotation angle for linear gradients.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Implement this method to register a gradient id. Then, use the id in the component color.[fillId](/Documentation/ApiReference/Common_Types/charts/ChartsColor/#fillId) field.
```

Comment:
Implement seems like the wrong word choice. They need to call the method, not implement it on their own. 

"Use in ... field" - I'd rather use a more direct verb "assign to ... field". Also, since we don't know which component we're talking about (could be any component), I would use indefinite article. 

```suggestion
Call this method to register a gradient ID. Assign the ID to a component's color.[fillId](/Documentation/ApiReference/Common_Types/charts/ChartsColor/#fillId) field.
```